tests/scripts/test_zeroize.gdb

```diff
@@ -43,7 +43,13 @@
 set confirm off
 
 file ./programs/test/zeroize
-break zeroize.c:100
+
+# Set breakpoint at the one line after the call to mbedtls_platform_zeroize()
+shell echo set \$zeroize_line = $(grep -n "mbedtls_platform_zeroize( buf, sizeof( buf ) )" ./programs/test/zeroize.c | cut -d: -f1) > zeroize_line.tmp
```

Comment:
This is still more fragile than it could be. If we change the spacing or rename `buf`, it won't match. If we change the code in such a way that there are multiple grep hits, it will cause the first one to be used (catching errors from gdb is impractical).

I don't know what the best solution is. Maybe take the _last_ hit for `mbedtls_platform_zeroize` (just the function name)?
```suggestion
shell echo set \$zeroize_line = $(grep -n "mbedtls_platform_zeroize" ./programs/test/zeroize.c | cut -d: -f1 | tail -n1) > zeroize_line.tmp
```